app/src/main/java/org/mozilla/fenix/share/ShareFragment.kt

```diff
@@ -86,6 +86,7 @@ class ShareFragment : AppCompatDialogFragment() {
 
         shareInteractor = ShareInteractor(
             DefaultShareController(
+                context = context!!,
```

Comment:
nit: `requireContext()` so we get better logs if this fails.

Reply:
@jonalmeida what better logs does `requireContext()` give vs. `context!!`? Just curious because I didn't know they were different!

Reply:
@sblatz requireContext, explicitly [throws an exception](http://androidxref.com/9.0.0_r3/xref/frameworks/support/fragment/src/main/java/androidx/fragment/app/Fragment.java#680) which gives better stack traces than `KotlinNullPointException` (imo).